src/android/Peripheral.java

```diff
@@ -408,31 +408,27 @@ public void onServicesDiscovered(BluetoothGatt gatt, int status) {
     }
 
     /**
-     * This enum lists all the BLE devices which dont need Auto connect but require few reconnect attempts when they face Gatt133 connection error
-     * Like, On Samsung Tab A7 , with Welch devices, there was random gatt 133 error while making a connection.
-     * Thus tried to reconnect in case of 133 failure which solves the problem.
+     * This enum lists all the BLE devices which face issues while we perform a retry after disconnection due to gatt error code (133)
      */
-    public enum AUTO_CONNECT_OFF_DEVICES {
-        WELCH_SC100("SC100"),
-        TNG_SCALE("TNG SCALE"),
-        WELCH_BP100("BP100");
+    public enum DEVICES_TO_ESCAPE_RETRY {
```

Comment:
We have changed this enum now and it will contain any devices which do not want to perform retry. For now we dont have any but can extend this code in future.